Platon/backend/tests/profile_management_test.py

```diff
@@ -102,7 +120,193 @@ def test_get_research_info_invalid(self):
+        actual_response = self.client.post("/api/profile/skills", data=data,
+                                           headers={'auth_token': valid_token})
+        self.assertEqual(expected_response, json.loads(actual_response.data))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertIsNone(Skills.query.filter(Skills.name == 'german').first())
+
+    def jobs_put_invalid(self):
+        valid_token = generate_token(1,TestConfig.SESSION_DURATION)
+        data = {
+            'id': 200,
+            'name': 'University Student'
+        }
+        expected_response = {
+            'error': 'Please give an appropriate job ID'
+        }
+        actual_response = self.client.post("/api/profile/jobs", data=data,
+                                           headers={'auth_token': valid_token})
+        self.assertEqual(expected_response, json.loads(actual_response.data))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertIsNone(Jobs.query.filter(
+            Jobs.name == 'University Student').first())
+
+    def skills_put_invalid(self):
+        valid_token = generate_token(2,TestConfig.SESSION_DURATION)
+        data = {
+            'id': 200,
+            'name': 'C++'
+        }
+        expected_response = {
+            'error': 'Please give an appropriate skill ID'
+        }
+        actual_response = self.client.post("/api/profile/skills", data=data,
+                                           headers={'auth_token': valid_token})
+        self.assertEqual(expected_response, json.loads(actual_response.data))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertIsNone(Skills.query.filter(
+            Skills.name == 'C++').first())
+
+    def jobs_delete_invalid(self):
+        valid_token = generate_token(1,TestConfig.SESSION_DURATION)
+        data = {
+            'id': 100
+        }
+        expected_result = {
+            'error': 'Job does not exist'
+        }
+        actual_response = self.client.delete("/api/profile/jobs", data=data,
+                                             headers={'auth_token': valid_token})
+        self.assertEqual(expected_result, json.loads(actual_response.data))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertIsNotNone(Jobs.query.filter(Jobs.id == 100).first())
+
+    def skills_delete_invalid(self):
+        valid_token = generate_token(1,TestConfig.SESSION_DURATION)
+        data = {
+            'id': 100
+        }
+        expected_result = {
+            'error': 'Skill does not exist'
+        }
+        actual_response = self.client.delete("/api/profile/skills", data=data,
+                                             headers={'auth_token': valid_token})
+        self.assertEqual(expected_result, json.loads(actual_response.data))
+        self.assertEqual(400, actual_response.status_code)
+        self.assertIsNotNone(Skills.query.filter(Skills.id == 100).first())
+
```

Comment:
Please write your tests into another class. Because it decreases the readability.

Reply:
I created new class in my last commit